app/controllers/calendars_controller.rb

```diff
@@ -0,0 +1,213 @@
+class CalendarsController < ApplicationController
```

Comment:
Was goign through your Heroku deployment - couldn't see anywhere where a coach could input their availability?

Reply:
It should be in the profile page

Reply:
Go to My account drop down, and then click view profile, then you will see Add availability option

Reply:
OH I see. Good job!